src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -136,15 +138,17 @@
 	public static final Item POGO_CLUB = getNull();
 	public static final Item METAL_BAT = getNull();
 	public static final Item SPIKED_CLUB = getNull();
-	
+	public static final Item KINGS_SCEPTER_WHITE = getNull();
+	public static final Item KINGS_SCEPTER_BLACK = getNull();
```

Comment:
I think white_kings_scepter and black_kings_scepter work better as names

Reply:
I think it is useful to keep it this way so that it is easy to find together. It matches the json files and png name, and if those were changed those files would not be located next to each other

Reply:
Okay, but I think the naming convention currently in place is more important than having the files alphabetically next to each other (also, you can still navigate between them pretty easily either way in the IDE)